### PR TITLE
Remembering pagination state for tx table

### LINKED ISSUE #293: Facilitate user navigation by keeping track of where they were
## Overview
The back button resets to the initial view, which is annoying. 

## Acceptance Criteria
- When the user checks a transaction and clicks the back button, they should be redirected to their previous view on the transaction table.

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #293

## Description
When you find a special transaction from the table and you follow the link and then come back to the table, the pagination is lost and you must start from the first page again which leads to a terrible user experience.

This PR makes the table save in local storage the last pagination state and recovers it every time the table is used again.

## Test scenarios
To recreate this new behavior you must:
- go to the [home page](https://deploy-preview-336--teloscan-stage.netlify.app/)
- change the current page and the page size
- follow the link of any of the listed transaction
- press the Back button of your browser.
- the table should recover the last pagination state

## Checklist:
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/components/TransactionTable.vue [hunk 1]
```diff
@@ -97,14 +99,29 @@ export default {
...
+            }
+        }
+
         this.onRequest({
             pagination: this.pagination,
         });
     },
     methods: {
         async onRequest(props) {
+            console.log('onRequest', props);
```
Comment (donnyquixotic) [inline src/components/TransactionTable.vue:119]: remove console log

**Review by donnyquixotic — CHANGES_REQUESTED** [review]
We should handle this in the router by incorporating query/filter params as part of routing (e.g.  `/<address>?page=3` )  for the appropriate pages to track pagination instead of using state. This would allow more depth of history tracking and avoid unintended behavior. For instance, If you navigate to a different page and click on an _address_ in the table, the page state is applied where it shouldn't be.

File: src/components/TransactionTable.vue [hunk 2]
```diff
@@ -97,14 +99,29 @@ export default {
         this.columns[6].label = this.$t('components.value_transfer');
     },
     mounted() {
+        // restoring a possible last pagination state
+        const paginationStaet = localStorage.getItem(STORAGE_KEY);
```
Comment (EJSG) [inline src/components/TransactionTable.vue:103]: typo in variable name

File: src/components/TransactionTable.vue (hunk 1 above)
Comment (EJSG) [inline src/components/TransactionTable.vue:119]: rm console log

File: src/components/TransactionTable.vue (hunk 2 above)
Comment (Viterbo) [reply src/components/TransactionTable.vue]: This code is no longer present

**Review by EJSG — APPROVED** [review]
nice, this works great

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**